readme.md

```diff
@@ -843,14 +843,14 @@ test(t => {
 	const a = /foo/;
 	const b = 'bar';
 	const c = 'baz';
-	t.ok(a.test(b) || b === c);
+	t.true(a.test(b) || b === c);
 });
 ```
 
 Will output:
 
 ```
-t.ok(a.test(b) || b === c)
+t.true(a.test(b) || b === c)
```

Comment:
The power-assert example right under this should be moved a bit to the right. The lines do not point to what is described.